Pages/questionPage/basicGardening/basicGardening.css

```diff
@@ -0,0 +1,38 @@
+@import url(../../../styles.css);
+
+:root {
+    --option-bg-color:#557C55;
+    --option-bg-color-hover:#496b49;
+}
+
+.question-container {
+    background-color: var(--background-color);
+    border-radius: 1rem;
+}
+
+.question-header {
+    justify-content: space-between;
+}
+
+.question {
+    margin: 2rem 0 5rem 0;
+}
+
+.question-option {
+    cursor: pointer;
+    color: var(--white-color);
+    background-color: var(--option-bg-color);
+    letter-spacing: 2px;
+    font-weight: 500;
+    border-radius: 1rem;
+    margin: 1.6rem 0;
+    padding: 1.6rem;
+}
+
+.question-option:hover {
+    background-color: var(--option-bg-color-hover);
+}
+
+.question-option:focus {
+    background-color: var(--primary-color2);
+}
```

Comment:
just a nitpick but adding a newline at the end of files is considered a good practice
https://thoughtbot.com/blog/no-newline-at-end-of-file

Reply:
Thanks for pointing that out👍